lib/components/DownloadsScreen/downloaded_items_list.dart

```diff
@@ -1,104 +1,168 @@
 import 'package:finamp/components/delete_prompts.dart';
 import 'package:finamp/services/finamp_settings_helper.dart';
 import 'package:flutter/material.dart';
+import 'package:flutter_gen/gen_l10n/app_localizations.dart';
+import 'package:flutter_riverpod/flutter_riverpod.dart';
 import 'package:get_it/get_it.dart';
 
 import '../../models/finamp_models.dart';
 import '../../services/downloads_service.dart';
 import '../album_image.dart';
 import 'item_file_size.dart';
 
-class DownloadedItemsList extends StatefulWidget {
-  const DownloadedItemsList({super.key});
+class DownloadedItemsTitle extends StatelessWidget {
+  const DownloadedItemsTitle({super.key, required this.title});
+
+  final String title;
+
+  @override
+  Widget build(BuildContext context) {
+    return SliverToBoxAdapter(
+      child: Padding(
+        padding: const EdgeInsets.only(left: 16, top: 8, right: 16),
+        child: Text(
+          title,
+          style: Theme.of(context).textTheme.headlineSmall,
```

Comment:
That still looks a bit large. Maybe try adding emphasis though font weight instead of size? That often looks sleeker :)

Reply:
Fair, I'll do some reading on text styling then.

Reply:
Done :white_check_mark: